### ISSUE TITLE
[Feature Request] Whois search

### ISSUE THREAD

**Issue by insxa** [FOSSBilling/FOSSBilling#1652]
Does fossbilling plans to integrate whois search ?
Like to enter word in search box and it shows all available domains, or registered to transfer.
Maybe some ajax or java script to load more ligt.
Or this can, should be integrated with theming, creting new theme with this option ?

**Comment by Anuril** [comment]
Some domain registrar adapters already have this implemented afaik.

**Comment by BelleNottelling** [comment]
> Some domain registrar adapters already have this implemented afaik.

No, the email & custom registrars use WHOIS to validate that a domain isn't owned yet, but as far as I can tell the feature request is specifically for a domain search / suggestion tool rather than only being able to enter a specific domain name and seeing if it's available or not & that doesn't exist in any capacity in FOSSBilling

**Comment by insxa** [comment]
I meant something like this: [link](https://console.webhosting24.com/mydomainchecker.php)
Its based on [hexonet ](https://marketplace.whmcs.com/product/1654-ispapi-high-performance-domainchecker-module)
![16728-ec052a76aad54a98ec3813efd4747d0b 1](https://github.com/FOSSBilling/FOSSBilling/assets/19369515/debbe68c-7517-48e2-91eb-8eb276eb2c10)

**Comment by notbasetwo** [comment]
If someone picked this up - It is probably a better idea to implement RDAP, given that WHOIS is now sunsetted and is likely to be deprecated in the future.

Some CCTLDs do not support RDAP yet, but they are adopting it slowly.
